content/developers/apps/authorizing-oauth-apps.md

```diff
@@ -272,7 +272,7 @@ The optional `redirect_uri` parameter can also be used for localhost URLs. If th
 
 For the `http://localhost/path` callback URL, you can use this `redirect_uri`:
 
-   http://localhost:1234/path
+    http://localhost:1234/path
```

Comment:
I'm not sure if the code block is the best formatting given the previous line, but it seems like that was the intention here but was missing a space

Reply:
You're probably right here, but while where at it, let's change this to be our usual code block style:

````suggestion
```
http://localhost:1234/path
```
````